### PR TITLE
fix: broken MyC spec after SWA release

### PR DESCRIPTION
**Pull request by anandaroop**
### Description

There's a spec that's been failing on CI ever since we [toggled the `AREnableSubmitArtworkTier2Information` flag](https://github.com/artsy/echo/pull/685/files) as part of the [SWA Tier 2 release](https://www.notion.so/artsy/SWA-Tier-2-Launch-f353c144ff2e473b8ecb859f1d3ca804) earlier today.

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Update a MyC spec that fails after an Echo change - Roop

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/Scenes/MyCollection/Screens/Artwork/Components/MyCollectionArtworkSubmissionStatus.tests.tsx [hunk 1]
```diff
@@ -50,6 +50,10 @@ describe("MyCollectionArtworkSubmissionStatus", () => {
   })
 
   it("display Submission status and In Progress when submission is in progress", () => {
+    __globalStoreTestUtils__?.injectFeatureFlags({
+      AREnableSubmitArtworkTier2Information: false,
+    })
```
Comment (anandaroop) [inline src/app/Scenes/MyCollection/Screens/Artwork/Components/MyCollectionArtworkSubmissionStatus.tests.tsx:55]: Here and below I set the feature flag back to its pre-launch state — I only did this for the two specs that [started failing](https://app.circleci.com/pipelines/github/artsy/eigen/58187/workflows/de6bc995-d331-49fa-bc27-978c8871950d/jobs/200688) after the [SWA Tier 2 release](https://www.notion.so/artsy/SWA-Tier-2-Launch-f353c144ff2e473b8ecb859f1d3ca804).

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (Update a MyC spec that fails after an Echo change - Roop)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 08438ed2175ad80679b2af450212205d7072d4c8
</p>

File: src/app/Scenes/MyCollection/Screens/Artwork/Components/MyCollectionArtworkSubmissionStatus.tests.tsx (hunk 1 above)
Comment (anandaroop) [reply src/app/Scenes/MyCollection/Screens/Artwork/Components/MyCollectionArtworkSubmissionStatus.tests.tsx:55]: Note that there is an entire [block](https://github.com/artsy/eigen/pull/10733/files#diff-f9b49cf53b5810399171f76d3fa6a4797b83ad3e4ebd15e76688e417d6d09634R94) of specs that test the condition where the flag is enabled.

Maybe those are the only specs needed now? I'll leave that up to @dariakoko

**Review by brainbicycle — APPROVED** [review]
thank you!

File: src/app/Scenes/MyCollection/Screens/Artwork/Components/MyCollectionArtworkSubmissionStatus.tests.tsx (hunk 1 above)
Comment (dariakoko) [reply src/app/Scenes/MyCollection/Screens/Artwork/Components/MyCollectionArtworkSubmissionStatus.tests.tsx:55]: I'll have a look, thanks @anandaroop!

**Pull request merged**